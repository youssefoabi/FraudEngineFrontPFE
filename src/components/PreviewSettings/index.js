import { GridItem } from '@cdiscount/backoffice-ui';
import React from 'react';

const PreviewSettings = () => <GridItem title="PrÃ©visualisation" />;

PreviewSettings.propTypes = {};

PreviewSettings.defaultProps = {};

export default PreviewSettings;
