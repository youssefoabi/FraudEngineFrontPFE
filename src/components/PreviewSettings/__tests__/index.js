import { itRendersCorrectly } from '@cdiscount/jest-utils';
import React from 'react';

import PreviewSettings from '../';

describe(PreviewSettings.name, () => {
  itRendersCorrectly(<PreviewSettings />);
});
