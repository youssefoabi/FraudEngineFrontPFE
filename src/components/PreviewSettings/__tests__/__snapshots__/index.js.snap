// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreviewSettings renders correctly 1`] = `
<h
  fullheight={false}
  title="PrÃ©visualisation"
/>
`;
